### ISSUE TITLE
Metadata discovery service - dataset metadata record required?

### ISSUE THREAD

**Issue by arbakker** [INSPIRE-MIF/helpdesk#35]
I am currently working on improving the INSPIRE compliance of the Dutch national CSW service. While digging into this issue I ran into some questions. 

Our [discovery service](https://nationaalgeoregister.nl/geonetwork/srv/eng/csw?request=GetCapabilities&service=CSW&version=2.0.2) is described by two metadata records:

- [service metadata record]( https://www.nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/deb81ebe-c7a7-42ba-bc21-f07cd348c3e8)
- [dataset metadata record ](https://www.nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/76327507-4be4-4f2a-8d0a-83ec36656177)

Validating the dataset metadata against the [ETF-validator](http://inspire.ec.europa.eu/validator/#start-tests) (Conformance Class 2: 'INSPIRE data sets and data set series interoperability metadata') resulted in these errors:

> The metadata record set has 1 record(s) with errors for this assertion.
XML document '76327507-4be4-4f2a-8d0a-83ec36656177.xml', record '76327507-4be4-4f2a-8d0a-83ec36656177': At least one keyword must have the title 'GEMET - INSPIRE themes, version 1.0'.

> The metadata record set has 1 record(s) with errors for this assertion.
XML document '76327507-4be4-4f2a-8d0a-83ec36656177.xml', record '76327507-4be4-4f2a-8d0a-83ec36656177': The spatial representation type shall be given using at least one element gmd:spatialRepresentationType/gmd:MD_SpatialRepresentationTypeCode.

These validation issues seem impossible to address since:

- there is no suitable INSPIRE Data Theme for this dataset (the total collection of metadata records in the CSW)
- there is not suitable spatialRepresentationType for this dataset see [codelist](https://standards.iso.org/iso/19139/resources
/gmxCodelists.xml#MD_SpatialRepresentationTypeCode)

This raises the question whether it is mandatory at all for INSPIRE compliance to publish dataset metadata of discovery services. I also looked at discovery services of Flanders, Luxemburg and Germany but found no dataset metadata records for discovery services. I could not find any documentation on guidelines describing in what situation publishing dataset metadata is mandatory and in what situation publishing service metadata is mandatory. 

So in short, is it required by INSPIRE to publish dataset metadata for a discovery service en where can I find the relevant documentation?

**Comment by arbakker** [comment]
@heidivanparys thanks for the detailed explanation why dataset metadata is not mandated by INSPIRE for discovery services! Much appreciated 👍
